<?xml version="1.0"?>
<launch>

    <node pkg="dvs_sync" type="rosbag_imu_fix" name="rosbag_imu_fix_node" respawn="false" output="screen">
        <param name="config_file" value="$(find dvs_sync)/config/utils.yaml"/>
    </node>

</launch>